src/daft-logical-plan/src/optimization/optimizer.rs

```diff
@@ -100,9 +100,6 @@ impl Default for OptimizerBuilder {
                         Box::new(UnnestScalarSubquery::new()),
                         Box::new(UnnestPredicateSubquery::new()),
                         Box::new(EliminateSubqueryAliasRule::new()),
-                        Box::new(SplitActorPoolProjects::new()),
-                        Box::new(DetectMonotonicId::new()),
-                        Box::new(ExtractWindowFunction::new()),
```

Comment:
The window function exprs are a little weird right now and may cause incorrect pushdowns, because the columns used in the "OVER" clause are not accounted for properly in the column dependencies. I would suggest keeping this rule here for now, until we make a fix for that.

Reply:
Gotcha yeah sounds good